components/modal/src/Modal.js

```diff
@@ -114,36 +115,38 @@ export const Modal = ({
 	if (typeof window !== 'undefined') {
 		return ReactDOM.createPortal(
 			<ModalContext.Provider value={{ state }}>
```

Comment:
I would wrap everything with `<GEL/>`. No reason to leave the ModalContext provider outside that I can see.
Having `<GEL/>` outside will be more consistent with everything else we do but in the end it won't make much difference.